apps/rush-lib/src/api/RushConfigurationProject.ts

```diff
@@ -303,4 +326,35 @@ export class RushConfigurationProject {
     }
     return isMain;
   }
+
+  private _setLocalDependencyProjects(dependencies: IPackageJsonDependencyTable = {}): void {
```

Comment:
`_addLocalDependencyProjects`? "set" implies that it's only expected to be called once.